docs/grover.rst

```diff
@@ -26,3 +115,7 @@ grove.amplification.grover
         :members:
         :undoc-members:
         :show-inheritance:
+
+.. [1] Nielsen, M.A. and Chuang, I.L. Quantum computation and quantum information. Cambridge
+       University Press, 2000. Chapter 6.
+.. [2] https://arxiv.org/pdf/quant-ph/9605043.pdf
```

Comment:
authors and name of paper for full citation?

Reply:
of course